tests/test_sequence.py

```diff
@@ -286,148 +286,171 @@ def pulses():
     return [rise, sweep, fall]
 
 
-def test_switch_device(test_devices, pulses):
+@pytest.fixture
+def sequences(devices, pulses):
 
-    # Device checkout
+    seqs = []
     seq = Sequence(reg, Chadoq2)
+    seq.declare_channel("ising", "rydberg_global")
+    seqs.append(seq)
+
+    seq = Sequence(reg, MockDevice)
+    seq.declare_channel("ising", "rydberg_global")
+    seqs.append(seq)
+
+    seq = Sequence(reg, devices[0])
+    seq.declare_channel("digital", "raman_global", "q1")
+    seqs.append(seq)
+
+    seq = Sequence(reg, devices[1])
+    seq.declare_channel("ising", "rmn_local", "q0")
+    seqs.append(seq)
+
+    seq = Sequence(reg, devices[2])
+    seq.declare_channel(name="ising", channel_id="rydberg_global")
+    seq.add(pulses[0], "ising")
+    seq.add(pulses[1], "ising")
+    seqs.append(seq)
+
+    seq = Sequence(reg, devices[0])
+    seq.declare_channel(name="ising", channel_id="rydberg_global")
+    seq.add(pulses[0], "ising")
+    seq.add(pulses[1], "ising")
+    seqs.append(seq)
+
+    seq = Sequence(reg, devices[2])
+    seq.declare_channel("ising", "rmn_local1", "q0")
+    seqs.append(seq)
+
+    seq = Sequence(reg, devices[2])
+    seq.declare_channel("ising", "rmn_local2", "q0")
+    seqs.append(seq)
+
+    seq = Sequence(reg, devices[2])
+    seq.declare_channel(name="ising", channel_id="rydberg_global")
+    seq.add(pulses[0], "ising")
+    seq.add(pulses[2], "ising")
+    seqs.append(seq)
+
+    return seqs
```

Comment:
I'm not sure about this, it makes it very hard to know which sequence is which when reading the tests. 
Why don't you make a function like this instead (NOT a fixture)?
```python
def init_seq(device, ch_name, ch_id, *pulses, initial_target=None):
    seq = Sequence(reg, device)
    seq.declare_channel(ch_name, ch_id, initial_target=initial_target)
    for pulse in pulses:
        seq.add(pulse, ch_name)
    return seq
```
This way you can create a new sequence inside your test function in one line. That way it's also clear what's contained in the sequence from the call to the function.